reconnecting-websocket.ts

```diff
@@ -380,8 +382,8 @@ export default class ReconnectingWebSocket {
                 }
                 this._debug('connect', {url, protocols: this._protocols});
                 this._ws = this._protocols
-                    ? new WebSocket(url, this._protocols)
-                    : new WebSocket(url);
+                    ? new WebSocket(url, this._protocols, wsOpts)
+                    : new WebSocket(url, wsOpts);
```

Comment:
You can't replace `wsOpts` by the protocols, this doesn't work.

```suggestion
                    : new WebSocket(url, null, wsOpts);
```